app/views/schools/cohorts/show.html.erb

```diff
@@ -0,0 +1,64 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-3"><%= @cohort.start_year %></h1>
+
+    <p class="govuk-body-l govuk-!-margin-bottom-8">
+    <% if @school_cohort[:induction_programme_choice] === "core_induction_programme" %>
+      Training using accredited materials.
+    <% else %>
+      Using a training provider funded by the DfE.
+    <% end %>
+    </p>
+
+    <ol class="app-task-list">
+      <li>
+        <h2 class="app-task-list__section">
+          Next steps
+        </h2>
+        <ul class="app-task-list__items">
+
+          <% if @school_cohort[:induction_programme_choice] === "core_induction_programme" %>
```

Comment:
Three equal signs? We aren't in js land here :D 

https://stackoverflow.com/a/4467823

Let's move this check to the model
```
def school_chose_cip?
  induction_programme_choice == "core_induction_programme"
end
```